libwacom/libwacom.c

```diff
@@ -144,6 +144,10 @@ get_bus_vid_pid (GUdevDevice  *device,
 		*bus = WBUSTYPE_I2C;
 		retval = TRUE;
 		break;
+	case 68:
+		*bus = WBUSTYPE_MEI;
+		retval = TRUE;
+		break;
```

Comment:
I can't find this one upstream yet, is it scheduled for any particular kernel release?

Come to think of it, IMO this should be in hex since that's what the upstream header uses (same goes for the existing statements)